ada_feeding/scripts/ada_planning_scene.py

```diff
@@ -690,23 +694,50 @@ def update_table_detection(self) -> None:
         detected_table_pose.pose.position.y += self.table_detection_offsets[1]
         detected_table_pose.pose.position.z += self.table_detection_offsets[2]
 
-        # Calculate the angular distance between the latest and default table quaternions
-        latest_table_quat = np.array(
+        # Store default and latest table orientations as transforms3d quaternions
+        default_table_quat = np.array(
```

Comment:
Nit: this doesn't need to be re-created every time this function is called, since it is constant. I'd suggest creating it once in `__init__` and then using it here.